completion/available/makefile.completion.bash

```diff
@@ -17,7 +19,7 @@ _makecomplete() {
   for f in "${files[@]}" ; do
     while IFS='' read -r line ; do
       targets+=("$line")
-    done < <(grep -oE '^[a-zA-Z0-9_-]+:([^=]|$)' "$f" | cut -d':' -f1)
+    done < <(grep -E -o '^[a-zA-Z0-9_-]+:([^=]|$)' "$f" | cut -d':' -f1)
```

Comment:
You'll see these around the PR, trying to standardize on putting the `-E`/`-F` **first** to make auditing usages a bit easier in the future.